helm/portieris/templates/default/policies.yaml

```diff
@@ -2,73 +2,80 @@
-    - name: "*.icr.io/armada/*"
-      policy:
-    - name: "*.icr.io/armada-worker/*"
-      policy:
-    - name: "*.icr.io/armada-master/*"
-      policy:
-    {{ end }}
-    # This policy prevents Portieris from blocking itself
-    - name: "{{ .Values.image.host }}/{{ .Values.image.image }}:{{ .Values.image.tag }}"
-      policy:
-    # This policy allows Portieris to use Hyperkube to configure your cluster. This policy must exist if you uninstall Portieris.
+    # This policy prevents Portieris from denying it own updates. Do not remove this policy or portieris may not function correctly.
+    - name: "{{ .Values.image.host | default "docker.io/ibmcom"  }}/{{ .Values.image.image }}:{{ .Values.image.tag }}"
+    # This policy allows Portieris to use Hyperkube to configure your cluster. This policy must exist if order to uninstall Portieris.
     - name: "{{ .Values.hyperkube.repository }}:{{ .Values.hyperkube.tag }}"
-      policies:
-  
 ---
 apiVersion: securityenforcement.admission.cloud.ibm.com/v1beta1
 kind: ImagePolicy
 metadata:
-  name: portieris-default-image-policy
+  name: default
   namespace: kube-system
   annotations:
     helm.sh/hook: post-install
     helm.sh/hook-weight: "1"
 spec:
    repositories:
-    # This policy allows all images to be deployed into this namespace. This policy prevents breaking any existing third party applications in this namespace.
-    # IMPORTANT: Review this policy and replace it with one that meets your requirements. If you do not run any third party applications in this namespace, you can remove this policy entirely.
+    # This permissive policy allows all images to be deployed into this namespace. 
+    # IMPORTANT: Review this policy and replace it with one that meets your requirements.
     - name: "*"
       policy:
     {{ if .Values.IBMContainerService }}   
-    # These policies allow all IBM Cloud Container Service images from the global and all regional registries to deploy in this namespace.
-    # IMPORTANT: When you create your own policy in this namespace, make sure to include these repositories. If you do not, the cluster might not function properly.
+    # These policies allow all IBM Cloud Container Service images to deploy in this namespace.
+    # IMPORTANT: When you create your own policy in this namespace, be sure to retain these policies. If you do not, the cluster might not update or function properly.
+    - name: "registry*.bluemix.net/armada/*"
+    - name: "registry*.bluemix.net/armada-worker/*"
+    - name: "registry*.bluemix.net/armada-master/*"
     - name: "*.icr.io/armada/*"
-      policy:
     - name: "*.icr.io/armada-worker/*"
-      policy:
     - name: "*.icr.io/armada-master/*"
-      policy:
+    - name: "icr.io/armada/*"
+    - name: "icr.io/armada-worker/*"
+    - name: "icr.io/armada-master/*"
```

Comment:
Do we need to have `policy:` here? I assume it would be null if not set, but then do we need it in the `*` case either?

Reply:
no, removed